stdlib/2and3/imghdr.pyi

```diff
@@ -1,16 +1,21 @@
-from typing import overload, Union, Text, BinaryIO, Optional, Any, List, Callable
-import sys
 import os
+import sys
+from typing import Any, BinaryIO, Callable, List, Optional, Protocol, Text, Union, overload
+
+class _ReadableBinary(Protocol):
```

Comment:
I thought `_ReadableBinary` was a better name given we're only interested in binary data here; after all it's image format checking.